metaflow/plugins/argo/argo_workflows.py

```diff
@@ -2731,7 +2745,7 @@ def _compile_sensor(self):
             )
 
         # Useful to paint the UI
-        trigger_annotations = {
+        {
```

Comment:
@trhodeos we have impending plans to move away from `one sensor -> one workflow template` mapping (to save resource costs) to `one sensor -> multiple workflow templates` which will break the expectations in this PR for `--only-event-sensor-json`. can you help us with the timelines you are shooting for this PR? happy to chat over zoom if that's helpful.

Reply:
depending on your use case, an immediate workaround could also be to override argo_client.py through the extensions such that rather than submitting the CRDs to kubernetes, it just prints them out to stderr.

Reply:
but regardless this change is in the right direction. can you help us with details on how you have tested this change? we should be able to test this PR ourselves some point next week.

Reply:
One potential solution is to only add a CLI arg `--output-json-directory-only` and just push all json blobs to that directory. This would essentially allow us to not be tied to the number of resources we generate (as in, adding more sensors is not a problem).

I like the argo_client.py idea, _though_ we'd still like to push directly to argo while we migrate off of using metaflow directly. Still possible, but may require a bit more thinking.

For context: we are trying to use metaflow to generate the resources, spit them out to a gitops repo which has ArgoCD hooked up to it, for better staging of deploys.

As for testing, I'll test locally tomorrow!